assets/js/frontend/single-product.js

```diff
@@ -34,16 +34,22 @@ jQuery( function( $ ) {
 
 			$tab.closest( 'li' ).addClass( 'active' );
 			$tabs_wrapper.find( $tab.attr( 'href' ) ).show();
-		})
-		.trigger( 'init' );
+		});
 
-	$( 'a.woocommerce-review-link' ).click( function() {
-		$( '.reviews_tab a' ).click();
+	$( 'body' )
```

Comment:
Don't need a `$( 'body' )` here again, group with the `.on( 'init', '.wc-tabs-wrapper, .woocommerce-tabs'....`